helpradar.wsp

```diff
@@ -0,0 +1,13 @@
+﻿
```

Comment:
Ik weet niet of deze dient te worden ingechecked, maar dat kunnen Windows gebruikers beter beoordelen